aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -71,13 +71,15 @@ Resources:
             Resource:
               - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/<%=CDO.dcdo_table_name%>"
               - !Sub "arn:aws:dynamodb:${AWS::Region}:${AWS::AccountId}:table/<%=CDO.gatekeeper_table_name%>"
-          # Write analysis events to Firehose.
+          # Write to our Firehose streams
           # TODO: Import resources into stack.
           - Effect: Allow
             Action:
             - 'firehose:PutRecord'
             - 'firehose:PutRecordBatch'
-            Resource: !Sub "arn:aws:firehose:${AWS::Region}:${AWS::AccountId}:deliverystream/analysis-events"
+            Resource:
+              - !Sub "arn:aws:firehose:${AWS::Region}:${AWS::AccountId}:deliverystream/analysis-events"
+              - !Sub "arn:aws:firehose:${AWS::Region}:${AWS::AccountId}:deliverystream/i18n-string-tracking-events"
```

Comment:
(Noting the `TODO` above) ideally we would create this delivery stream as a resource in the application stack and add a stack reference to the resource's ARN directly, but if it's already been manually created it's fine to just make a manual reference here to unblock this work and we can follow up on importing the resource later.